electron/renderer/src/index.js

```diff
@@ -1,7 +1,26 @@
 import React from 'react'
 import { render } from 'react-dom'
+import { createStore } from 'redux'
+import { Provider } from 'react-redux'
+
+import { addAccount, switchAccount } from './actions'
+import appStore from './reducers'
+import { loadState, saveState } from './lib/localStorage'
+
+const persistedState = loadState()
+
+const store = createStore(
+  appStore,
+  persistedState
+)
+
+store.subscribe(() => {
+  saveState({
+    accounts: store.getState().accounts
```

Comment:
Annotation: Here we could use reselect or lodashs memoize to reduce the writes to the local storage by only calling the saveState when the state for the account actually changed.